cql/src/main/java/org/apache/cassandra/stargate/transport/internal/Server.java

```diff
@@ -284,6 +282,7 @@ public void register(Event.Type type, Channel ch) {
       groups.get(type).add(ch);
     }
 
+    @SuppressWarnings("FutureReturnValueIgnored")
     public void send(Event event) {
       groups.get(event.type).writeAndFlush(new EventMessage(event));
```

Comment:
https://errorprone.info/bugpattern/FutureReturnValueIgnored

I'm a bit annoyed with that one. It's a perfectly valid issue, but it often happens in cases like this where the API was not designed to propagate an async error. The method would have to return a future.